ozaria/site/components/teacher-dashboard/BaseMyClasses/components/ClassSummaryRow.vue

```diff
@@ -217,12 +219,33 @@
     align-items: center;
   }
 
+  .class-hover-area {
+    width: 100%;
+    &:hover {
```

Comment:
can we remove the previous hover code too?

Reply:
@mrfinch It's not clear for me what code to remove.
The `border` was removed from line 225, but we still need the `cursor: pointer.`